src/components/modules/DetailedPage/ProductHeader/ProductHeader.tsx

```diff
@@ -34,7 +34,7 @@ export default function ProductHeader({
           <HeaderTitle>{title}</HeaderTitle>
           <HeaderSubtitle>{subtitle}</HeaderSubtitle>
           <Row>
-            {tags.map((tag: string, index: number) => (
+            {tags.map((tag, index) => (
```

Comment:
typescript already knows the types of these implicitly since you've mentioned in the interface that tags only accepts an array of strings aka 
```
interface ProductHeaderProps {
  ...
  tags: string[];
  ...
}
```